.github/workflows/esti.yaml

```
@@ -1184,13 +1184,22 @@ jobs:
       AZURE_CLIENT_ID: ${{ secrets.AZURE_CLIENT_ID }}
       AZURE_CLIENT_SECRET: ${{ secrets.AZURE_CLIENT_SECRET }}
       AZURE_TENANT_ID: ${{ secrets.AZURE_TENANT_ID }}
+      COSMOSDB_DATABASE: esti-db
+      COSMOSDB_ACCOUNT: esti-e2e-tests
+      COSMOSDB_CONTAINER_PREFIX: ${{ github.run_number }}
```

```suggestion
      COSMOSDB_CONTAINER_PREFIX: azure-adls-gen2-${{ github.run_number }}
```
unless there's a low limit on the length of a CosmosDB container name, of course.  Having a prefix makes it easier to identify the container, and may also help if we need _other_ CosmosDB containers elsewhere.

---

Not sure this is actually necessary, the run log shows explicitly the env variables and it's quite easy to see which container belongs to which job.

---

Thanks anyway! Going through logs works for going from a run to a container. I was worried about going from a container to a step, there we don't have logs.